src/components/tasks/card/index.tsx

```diff
@@ -264,6 +276,7 @@ const Card: FC<Props> = ({
         const response = updateTask({
             task: data,
             id: cardDetails.id,
+            ...(isDevEnabled && { isDevEnabled: true }),
         });
```

Comment:
`isDevEnabled: true`  can't we only have passed instead of `...(isDevEnabled && { isDevEnabled: true })` , just asking

Reply:
we can however I thought why to un necessary pass it if its not found.